python_client/pyrefinebio/http.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+def get(endpoint, params=None):
+    url = ""
+
+    if endpoint.startswith("http"):
+        url = endpoint
+    else:
+        url = "https://api.refine.bio/v1/" + endpoint + "/"
```

Comment:
I like the idea that we force HTTPS, but I don't think this is quite the right way to do it.

I also think that if you want to allow the user to pass a URL instead an endpoint, we should just have two functions for that. Maybe `get()` and `get_by_endpoint()`. Then you can just have `get_by_endpoint()` call `get()` with the right URL.

Reply:
This was just a temporary thing to deal with calling the `next` urls when iterating through the pagination because they had the full url and I originally wrote this to use just the endpoints.
I'll split it into the two functions during the next commit though!